docker/freebsd-common.sh

```diff
@@ -6,57 +6,14 @@ set -euo pipefail
 # shellcheck disable=SC1091
 . freebsd-arch.sh
 
-export BSD_ARCH=
+export FREEBSD_ARCH=
 case "${ARCH}" in
     x86_64)
-        BSD_ARCH=amd64
+        FREEBSD_ARCH=amd64
         ;;
     i686)
-        BSD_ARCH=i386
+        FREEBSD_ARCH=i386
         ;;
 esac
 
-# we prefer those closer in geography to the US. they're triaged in
-# order of ease of use, reliability, and then geography. the mirror
-# list is at https://docs.freebsd.org/en/books/handbook/mirrors/.
-# these mirrors were known to work as of 2022-11-27. this does
-# not include any mirrors that are known to be rate-limited or
-# commercial.
-export BSD_HOME=(
```

Comment:
Since this is only used in `freebsd.sh`, and these mirrors aren't the same as the packagesite mirrors, I moved them to `freebsd,sh`.